src/renderer/account/components/AccountPanel/Breakdown/TotalValue/TotalValue.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import classNames from 'classnames';
+import { number, string } from 'prop-types';
+
+import formatCurrency from '../../../../util/formatCurrency';
+import styles from './TotalValue.scss';
+
+export default class TotalValue extends React.PureComponent {
+  static propTypes = {
+    className: string,
+    total: number.isRequired,
+    currency: string.isRequired
+  };
+
+  static defaultProps = {
+    className: null
+  };
+
+  render() {
+    const { className, total, currency } = this.props;
+
+    return (
+      <div className={classNames(styles.totalValue, className)}>
+        <div className={styles.value}>{formatCurrency(total, currency)}</div>
```

Comment:
If there no balance, it should say `No funds avaiilable`

Reply:
That's being tracked under a zero-state issue that hasn't been done yet: https://github.com/nos/client/issues/408